packages/web/src/components/user-list/UserList.tsx

```diff
@@ -14,11 +14,17 @@ import { UserListItemSkeleton } from './UserListItemSkeleton'
 const SCROLL_THRESHOLD = 400
 const DEFAULT_SKELETON_COUNT = 10
 
+type UserListDataItem = ID | { userId: ID; balance?: number }
+
+function getUserId(item: UserListDataItem): ID {
+  return typeof item === 'object' && item !== null ? item.userId : item
+}
```

Comment:
is there a more secure way to do this? perhaps with a type guard for the object or something?